antlr-kotlin-runtime/src/commonMain/kotlin/com/strumenta/kotlinmultiplatform/Arrays.kt

```diff
@@ -1,9 +1,9 @@
 package com.strumenta.kotlinmultiplatform
 
-object Arrays {
-  fun equals(a: Array<*>?, b: Array<*>?): Boolean =
+public object Arrays {
```

Comment:
Isn't public the default?

Reply:
It is, but with [explicit API mode](https://github.com/Kotlin/KEEP/blob/master/proposals/explicit-api-mode.md) you must explicitly state the visibility of components.

Reply:
Understood, make sense